src/main/java/seedu/address/logic/Logic.java

```diff
@@ -33,6 +35,12 @@
     /** Returns an unmodifiable view of the filtered list of persons */
     ObservableList<Applicant> getFilteredPersonList();
```

Comment:
Would it be better if we change this method to `getFilteredApplicantList`? I feel it's clearer in case we would want to expand on having different types of persons (like an interviewer) along with an applicant instead

@SethCKL any thoughts as well?